opensearch-operator/api/v1/opensearch_types.go

```diff
@@ -84,6 +84,27 @@ type InitHelperConfig struct {
 	Version    *string                     `json:"version,omitempty"`
 }
 
+type ProbesConfig struct {
```

Comment:
Can you please add some unit tests for this change in https://github.com/opensearch-project/opensearch-k8s-operator/blob/main/opensearch-operator/pkg/builders/cluster_test.go.

Reply:
Probes unit tests were not there earlier, let me try to add some tomorrow.

Reply:
Added tests to cover 100% changed lines.